tools/build.py

```diff
@@ -448,6 +467,7 @@ def run_checktest(options):
 
     # Run tests.
     if options.run_test:
+        build_napi_test_module(options)
```

Comment:
This should not be after the `run_test`.

Reply:
What if a compilation error occurred in napi test module? Shall it block main procedure? 🤔

Reply:
https://github.com/Rokid/ShadowNode/pull/243/files/00c9ee210b3837abadf8ca30c65d2b691e2cd807#diff-d504d105fb41328f6722f4e5c8432a96L421 Put this line here?